parameter_files/WebParameters_2dii.yml

```diff
@@ -1,7 +1,7 @@
 default:
     paths:
-        project_location_ext: C:/Users/clare/Desktop/TestGit/PACTA_analysis/
```

Comment:
We should probably make this a GitIgnore file? Or is there an option to keep a "template" version of this and changes are ignored?

Reply:
I've been wondering how best to that kind of thing for a while now. Have been talking to Jackson recently, be also did not know. I mean, we can add it to gitignore, but then we will have to assume that everyone has this file installed... I guess we could stop tracking it and see what happens when the next person pulls code from the repo...

Reply:
@Clare2D since the discussion on this topic does not seem overly promising neither on stackoverflow nor on our slack channel, we could just go for the same logic as in the normal Pacta code and copy the parameterfiles into the 10_Parameter_File folder... that would at least be consistent